### PR TITLE
mergepbx 0.9 (new formula)

### PR DESCRIPTION
**Pull request by theoriginalgri**
script for merging XCode project files in git

### CODE REVIEW

File: Library/Formula/mergepbx.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  homepage "https://github.com/simonwagner/mergepbx"
+  url "https://github.com/simonwagner/mergepbx/archive/v0.9.tar.gz"
+  version "0.9"
+  sha256 "d0d859a902357f8059fca0c5fb061a9072b82cb7cce4f38bd9eb2c3198fce003"
+
+  def install
+    system "./build.py"
+    bin.install "mergepbx"
+  end  
+end
```
Comment (mikemcquaid) [inline Library/Formula/mergepbx.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/mergepbx.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Mergepbx < Formula
+  homepage "https://github.com/simonwagner/mergepbx"
+  url "https://github.com/simonwagner/mergepbx/archive/v0.9.tar.gz"
+  version "0.9"
```
Comment (mikemcquaid) [inline Library/Formula/mergepbx.rb]: Pretty sure this isn't needed.

**Comment by theoriginalgri** [comment]
I added the test step as you wished. The command has no version output, nor can I create a xcode project on the fly, so the only chance was the help output.

Your were right, the version was not needed and is auto-determined.

**Comment by mikemcquaid** [comment]
> nor can I create a xcode project on the fly, so the only chance was the help output.

It'd be good if we could embed some sample data to write out and/or download it as a resource.

**Comment by theoriginalgri** [comment]
I hope it looks better now :)

**Comment by mikemcquaid** [comment]
Thats great, thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**